confidant/schema/blind_credentials.py

```diff
@@ -0,0 +1,71 @@
+import attr
+import toastedmarshmallow
+from marshmallow import fields
+
+from confidant.schema.auto_build_schema import AutobuildSchema
+
+
+@attr.s
+class BlindCredentialResponse(object):
+    id = attr.ib()
+    name = attr.ib()
+    cipher_version = attr.ib()
+    cipher_type = attr.ib()
+    data_key = attr.ib()
+    revision = attr.ib()
+    enabled = attr.ib()
+    modified_date = attr.ib()
+    modified_by = attr.ib()
+    metadata = attr.ib(default=dict)
+    credential_keys = attr.ib(default=list)
+    credential_pairs = attr.ib(default=dict)
+
+    @classmethod
+    def from_blind_credential(
+        cls,
+        credential,
+        include_credential_keys=False,
+        include_credential_pairs=False,
+        include_data_key=False,
+    ):
+        ret = cls(
+            id=credential.id,
+            name=credential.name,
+            cipher_version=credential.cipher_version,
+            cipher_type=credential.cipher_type,
+            metadata=credential.metadata,
+            revision=credential.revision,
+            enabled=credential.enabled,
+            modified_date=credential.modified_date,
+            modified_by=credential.modified_by,
```

Comment:
New to `attr`, but should we add a default for `data_key` in case the flag doesn't get passed in with `true`?  The others `credential_keys` and `credential_pairs` include defaults.